dev-app/src/main/java/com/uid2/dev/ui/MainScreenViewModel.kt

```diff
@@ -51,11 +51,15 @@ sealed interface MainScreenState : ViewState {
 class MainScreenViewModel(
     private val api: AppUID2Client,
     private val manager: UID2Manager,
+    isEUID: Boolean,
```

Comment:
Nit: Since this is dev code, I don't feel strongly at all, but just to call out that for the vast majority of users, the default will be UID2. Therefore it would make sense to have this inverted where `true` is the default, and therefore `isUID2`.